### PR TITLE
Feature/capability

### PR DESCRIPTION
**Pull request by e-mccoy**

### CODE REVIEW

File: Doras/src/app/data.service.ts [hunk 1]
```diff
@@ -53,6 +57,30 @@ export class DataService {
     return this.http.get<Training[]>('/api/trainingByJobId/?jobId=' + jId);
   }
 
+  public newCapability(newCapability) {
+    return this.http.post('/api/newCapability', newCapability);
```
Comment (micalexis) [inline Doras/src/app/data.service.ts]: make endpoints RESTful

File: Doras/src/app/delete-capability/delete-capability.component.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  ngOnInit() {
+    this.data.getCapabilities().subscribe((c) => {
+      this.capabilities = c;
+    });
+  }
+
+  onSubmit(form: NgForm) {
+    if (form.valid && this.invalid !== true) {
+      this.data.deleteCapability(form.value.cap).subscribe((c) => {
+        window.location.reload();
```
Comment (micalexis) [inline Doras/src/app/delete-capability/delete-capability.component.ts]: why we doing this?

File: Doras/src/app/models/capability.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export class Capability {
     id: number;
-    name: string;
+    capName: string;
     leadId?: number;
-    capId?: number;
+    jfId?: number;
```
Comment (micalexis) [inline Doras/src/app/models/capability.ts]: should this be optional? is it nullable in the database?

File: ExpressAPI/app.js [hunk 4]
```diff
@@ -83,6 +83,18 @@ app.get('/band', function(req, res) {
...
 })
 
+function getCapLeads(capLeadsReadyFn) {
+      db.getCapLeads(function(rows) {
+          capLeads = rows
+          capLeadsReadyFn()
+      })
+  }
+
+  app.get('/getCapLeads', function(req, res){
```
Comment (micalexis) [inline ExpressAPI/app.js:91]: make RESTful

**Review by micalexis — CHANGES_REQUESTED** [review]
make changes
squash commits

**Review by gordonkelly26 — APPROVED** [review]
I approve this

**Pull request merged**